scss/_patterns_notifications.scss

```diff
@@ -1,8 +1,8 @@
 @import 'settings';
-$notification-content-vert-padding: $spv-inner--small;
 $notification-content-icon-space: 2 * $sph-inner + map-get($icon-sizes, default);
-$notification-icon-vert-offset: 0.6rem;
-$notification-text-margin-bottom: 0.6rem;
+$borderless-notification-icon-vert-offset: $spv-nudge + 0.5 * (map-get($line-heights, default-text) - map-get($icon-sizes, default));
```

Comment:
_What in the world?_

Reply:
We need some way for documenting this. We should not expect other devs, or even ourselves, to reverse engineer these expressions later.

It took me about half an hour to (I hope) get what this expression does. I needed to debug  it in the browser (play with the values), get into spacing settings (to get the numbers) and almost draw this thing in my mind to visualize it. Now I kinda understand that `$sp-nudge` is aligning the icon with the content text box, and the rest is aligning the icon in the middle of the text baseline.

Sure, it boils down to relatively simple math and it's partially SCSS that makes it look scary (with all the vars and map-gets), but we are also not helping. Variable names are not always helpful, and specifically it's not always clear what exactly we are trying to calculate.

I don't know how we can easily simplify it, other than having a code comment every time we do something like this.

Reply:
I'm very open to ideas on how to make this more readable. I'll unpack it in comments for now. ALl I'm trying to do is make it robust - so if you change one of the value this positioning relies on, it will stay properly aligned.